meta/inspiredminds/contao-search-and-replace/de.yml

```
@@ -0,0 +1,4 @@
+de:
+    title: Search & Replace
+    description: >
+        Mit dieser Extension können im Backend Zeichenfolgen in der Datenbank suchen und ersetzt werden.
```

```suggestion
        Mit dieser Extension können im Backend Zeichenfolgen in der Datenbank gesucht und ersetzt werden.
```

Grammar, or maybe use
```Mit dieser Extension können Sie im Backend Zeichenfolgen in der Datenbank suchen und ersetzen.```